src/app/common/transactions/broadcast-transaction.ts

```diff
@@ -25,7 +25,7 @@ export async function broadcastTransaction(options: BroadcastTransactionOptions)
     const isValidTxId = validateTxId(response.txid);
     if (isValidTxId)
       return {
-        txId: response,
+        txId: response.txid,
```

Comment:
@kyranjamie can you confirm this change is ok? Things were broken in the test-app bc it was expecting an txId as a string. I am thinking maybe you meant it to be just the `txid`?

Reply:
Yeah, this makes sense. This looks like another issue caused by the stacks/tx update.